### ISSUE TITLE
Pointer.combine problem

### ISSUE THREAD

**Issue by trou** [airbus-seclab/bincat#13]
* X1 is a register with a S0x0 value
* an instruction sets x1[0,31] to G0

=> Bottom value :(

**Comment by trou** [comment]
The test I pushed passes with "sp" as a regular register, but fails with sp as a stack pointer:
```
$ python -m pytest -k stack_com test_armv8_isn.py 
[...]
E       AssertionError: test_stack_combine: (x3)
E       =========================
E          0:	d2800003 	mov	x3, #0x0                   	// #0
E          4:	910003e0 	mov	x0, sp
E          8:	910003e1 	mov	x1, sp
E          c:	cb010002 	sub	x2, x0, x1
E         10:	f9000002 	str	x2, [x0]
E         14:	b9400003 	ldr	w3, [x0]
E         18:	91040063 	add	x3, x3, #0x100
E       =========================
E       - cpu   :  x3 = 00000100
E       + bincat:  x3 = 00000000  Value(b, 0000000000000000,_=000000000000000F ! 0000000000000000)
E       =========================
```
`x3` is actually `0x_`

**Comment by trou** [comment]
Fixed in v1.0rc, by getting rid of the "stack" region
